__tests__/get.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection"); //imports db again from connection
+const seed = require("../db/seeds/seed"); //imports seeding
+const request = require("supertest"); //imports supertest (a library which allows http express requests)
+const data = require("../db/data/test-data/index"); //imports test data
+const app = require("../app"); //imports app has all the application and routes in
+
+beforeEach(() => seed(data)); // sets up function to define what happens before each test, here we populate the database with data before testing
+afterAll(() => db.end()); //used to close connection after each test
+
+describe("GET /api/topics", () => {
+  test("should return an array of objects containing slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.topics.length).toBe(3); // checks topics should return array of 3
+        res.body.topics.forEach((topic) => {
+          // loop  through each topic in the res checks properties named slug and description
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
+      });
+  });
+  describe("GET /api/topics/invalidendpoint", () => {
+    test("should return a 404 status and the error message", () => {
+      return request(app)
+        .get("/api/topics/invalidendpoint")
+        .expect(404)
+        .then((res) => {
+          expect(res.body.msg).toEqual("Request not found");
+        });
+    });
+  });
+});
```

Comment:
This test is checking for '/api/topics/somethingelse' however currently we haven't set up any endpoint with this path - consider using '/api/somethingelse' at this point and this will cover any invalid request to your api.